app/assets/javascripts/approve_leave_confirm.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+  $(".approve_leave_btn").click(function(){
+    var leave_id = $(this).val();
+    $('#approve_leave_modal').modal('show');
+    $("#approve_leave_yes").click(function(){
+      $.ajax({
+        url: '/approve_leave',
+        type: 'POST',
+        dataType: 'script',
+        data: { leave_request_id: leave_id }
```

Comment:
Identifier 'leave_request_id' is not in camel case.<br>Identifier 'leave_id' is not in camel case.